assets/base.css

```diff
@@ -2463,13 +2485,19 @@ product-info .loading-overlay:not(.hidden) ~ *,
   background: rgba(var(--color-foreground), 0.5);
 }
 
+@media screen and (max-width: 989px) {
+  menu-drawer ~ .header__icons .header__icon--account {
+    display: none;
+  }
+}
```

Comment:
I don't think this ever does anything. The only time we use `menu-drawer` is for filters on collection pages. 
So this will never be a sibling of the `header__icons` element.

Reply:
Honestly, I did not touch this code, it was here before. But it seems you are right. Let's remove it but keep on eye on it.